docker/Dockerfile-clang

```diff
@@ -49,6 +49,7 @@ RUN useradd -m espresso && usermod -a -G www-data espresso
 USER espresso
 ENV HOME="/home/espresso"
 # install Python3 packages locally
-RUN pip3 install --user requests autopep8==1.3.4 pycodestyle==2.3.1 pylint==2.2.2 astroid==2.1.0 isort==4.3.4 cython==0.26 cmake==3.10.3 cmake-format==0.6.9 pre-commit
 ENV PATH="${PATH}:${HOME}/.local/bin"
+RUN pip3 install --upgrade pip
+RUN pip install --user requests autopep8==1.3.4 pycodestyle==2.3.1 pylint==2.2.2 astroid==2.1.0 isort==4.3.4 cython==0.26 cmake==3.10.3 cmake-format==0.6.9 pre-commit
```

Comment:
pip3?

Reply:
no, you want to use the updated pip that has now ```3```